packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -104,7 +99,7 @@ public void onMethodCall(MethodCall call, Result result) {
                 new String[] {
                   Manifest.permission.CAMERA, Manifest.permission.READ_EXTERNAL_STORAGE
                 },
-                REQUEST_PERMISSIONS);
+                REQUEST_CAMERA_PERMISSIONS);
             break;
           }
           activity.startActivityForResult(
```

Comment:
Add a TODO here that the Camera should also be switched to use the native camera?